include/oneapi/dpl/experimental/kt/esimd_radix_sort.h

```diff
@@ -31,6 +31,15 @@ template <bool _IsAscending, std::uint8_t _RadixBits, typename _KernelParam, typ
 sycl::event
 radix_sort(sycl::queue __q, _Range&& __rng, _KernelParam __param)
 {
+    static_assert(_RadixBits == 7 || _RadixBits == 8);
```

Comment:
I would suggest leaving only 8 for now. 7 has not been verified with the majority of configurations.

Reply:
Fixed